.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+on:
+  # Trigger the workflow on push to the main branch of pull request
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+      - run: npm ci
+        env: 
+          MONGOMS_DISABLE_POSTINSTALL: 1
+      - run: npm run generate
+      - run: npm run build
+  audit:
+    name: Audit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+      - run: npm audit --audit-level=high --only=prod
```

Comment:
imo audit doesn't need to be run on every PR, should be run local every once in a while

Reply:
or author of PR should post results of running it locally if they update/add deps

Reply:
What about running it on push to main? It wouldn't block PRs, would be automatic, and would be good info to know